spk/nzbget/src/nzbget.conf

```diff
@@ -85,8 +85,14 @@ ScriptDir=/usr/local/nzbget/share/nzbget/scripts
 
```

Comment:
This file was copied directly from the tarball for 15.0. It must have preserved its permissions from there - I will fix that.